src/fetcher.ts

```diff
@@ -1694,7 +1697,7 @@ export default class Fetcher implements CallbackifyInterface {
     kb.add(responseNode, this.ns.http('statusText'),
     kb.rdfFactory.literal(response.statusText), responseNode)
 
-    if (!options.resource.value.startsWith('http')) {
+    if (!response.headers) {
```

Comment:
Yeah, I wondered about that. I don't see any harm in saving headers from "dumb clients" (e.g. that don't send wac-allow), but maybe there was a reason to prevent saving them?